CONTRIBUTING.md

```diff
@@ -204,3 +193,32 @@ particular order:
 - The root folder of the workspace
 - All other workspace roots
 - Every include path specified by the [DML Compile commands](README.md#dml-compile-commands) under the related module
+
+### <a id="device contexts"></a>Device Contexts
+Semantic analysis in the DLS is based around 'device contexts', essentially
```

Comment:
For later: some of the text here should probably also be present in the extension wiki https://github.com/intel/vscode-simics/wiki